modules/cms-kit/src/Volo.CmsKit.Admin.Web/Pages/CmsKit/GlobalResources/Index.cshtml

```diff
@@ -17,8 +18,15 @@
     PageLayout.Content.MenuItemName = CmsKitAdminMenus.GlobalResources.GlobalResourcesMenu;
 }
 
+@section styles{
+    <abp-style type="typeof(CodemirrorStyleContributor)" />
+}
+
 @section scripts {
     <abp-script-bundle>
+        <abp-script type="typeof(CodemirrorScriptContributor )"/>
+        <abp-script src="/libs/codemirror/mode/css/css.js"/>
+        <abp-script src="/libs/codemirror/mode/javascript/javascript.js"/>
```

Comment:
Why they're added here manually instead of adding to ScriptContributor ?

Reply:
assume any people who wants to use `codemirror` in other place, but they don't need 'js' or 'css' mode.   Maybe they just want 'csharp' or 'java'. If we add all "mode"s to `ScriptContributor`,  it will cause final bundle size lager.  So leave this option(add manually) to developer is better.